crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -134,7 +134,7 @@ def adminrepocheck
             repos: {}
           }
           ret[:os][:repos][admin_architecture.to_sym] = {
-            missing: ["SUSE Linux Enterprise Server 12 SP2"]
+            missing: ["SUSE-Linux-Enterprise-Server-12-SP2"]
```

Comment:
Um... should it actually be `SLES12-SP2-Pool` and `SLES12-SP2-Updates`? 
Both of them until we have a way to finding out which one is missing